libs/expo/betterangels/src/lib/screens/AddEditClient/ContactInfo.tsx

```diff
@@ -34,6 +36,12 @@ export default function ContactInfo(props: IContactInfoProps) {
 
   const isContactInfo = expanded === 'Contact Info';
 
+  useEffect(() => {
+    if (phoneNumber === '') {
+      setValue('phoneNumber', null);
+    }
+  }, [phoneNumber]);
+
```

Comment:
I created this because of this error here:

`[ApolloError: Variable '$data' got invalid value '' at 'data.phoneNumber'; Invalid phone number format: (1) The string supplied did not seem to be a phone number.]`